src/main/scala/inox/solvers/smtlib/SMTLIBParser.scala

```diff
@@ -224,7 +224,9 @@ trait SMTLIBParser {
       val MapType(from, to) = fromSMT(sort)
       FiniteMap(Seq.empty, d, from, to)
 
-    case _ => throw new MissformedSMTException(term, "Unknown SMT term")
+    case _ => throw new MissformedSMTException(term,
+      " Unknown SMT term of class: " + term.getClass
```

Comment:
Nit: is the extra space before `Unknown` expected?

Reply:
No, I was concatenating something before it for debugging, thanks!